Business Rules/Create Auto-Assign Incident to Support Group/Auto-Assign Incident to Support Group.js

```diff
@@ -0,0 +1,14 @@
+// Situation: When a new incident is created, automatically assign it to the appropriate support group based on the categorization.
+
+(function executeRule(current, previous /*, g*/) {
+    if (current.category == 'Hardware' && current.subcategory == 'Laptop') {
+        current.assignment_group = 'Hardware Support';
```

Comment:
Dot-walking to get and set values is not recommended and will not always behave as expected. Use `current.getValue(field)` and `current.setValue(field,value)` to appropriately access and set these values.

Additionally, setting assignment group by it's display field (name) is also not best practice.